cmake/TivaCMake/FindCodeComposerStudio.cmake

```diff
@@ -33,22 +33,29 @@ The following cache variables may also be set:
   The executable that runs the UniFlash tool, used for flashing microcontrollers
 #]========================================================================]
 # A guide for writing find modules: https://cmake.org/cmake/help/v3.17/manual/cmake-developer.7.html
+message(FATAL_ERROR "system name is ${CMAKE_HOST_SYSTEM_NAME}")
 if(NOT CodeComposerStudio_FOUND)
   # Gather directory names to search
   file(GLOB CodeComposerStudio_ROOTS
-    $ENV{HOME}/ti/ccs* $ENV{HOME}/ccs* /opt/ccs* /opt/ti/ccs*)
+    $ENV{HOME}/ti/ccs* $ENV{HOME}/ccs* /opt/ccs* /opt/ti/ccs* /Applications/ti/ccs* )
 
   # Find the code composer studio executable
   find_program(CodeComposerStudio_EXECUTABLE
     NAMES ccstudio
-    HINTS ${CodeComposerStudio_ROOTS}
+	HINTS ${CodeComposerStudio_ROOTS}
     PATH_SUFFIXES ccs/eclipse
     NO_DEFAULT_PATH
     )
 
   # Get the root directory
-  get_filename_component(CodeComposerStudio_EXEPATH ${CodeComposerStudio_EXECUTABLE} DIRECTORY)
-  get_filename_component(CodeComposerStudio_ROOT_DIR ${CodeComposerStudio_EXEPATH}/../.. ABSOLUTE)
+  
```

Comment:
Add a comment about how On MacOS CCstudio is in an app bundle, so the executable is few more directories deep